internal/cueutils/convert/to_cli/from_testapi.cue

```diff
@@ -0,0 +1,18 @@
+package present
+
+import "snyk.io/dragonfly/pkg/schemas:testapi"
+
+input: _
+
+input: {
+    test: testapi.#TestResource
+    findings: [...testapi.#FindingResource]
+}
+
+// This should align with
+// LocalFinding definition
```

Comment:
I think Option 2 would mean we are detached from the dragonfly types. Which I would like to avoid to be honest.

Reply:
Is there maybe an **Option 3** that makes use of any of the [encodings in cue](https://pkg.go.dev/cuelang.org/go/encoding@v0.10.0) instead of defining these cue types in code, we should be able to import them from something that comes from dragonfly. 
So that dragonfly would still be the single source of truth.

Reply:
I think we'd want to do that either way (reference the original Dragonfly types)!

With Option 1, you'd import the test API types into a new Typespec model defining the output schema.

With Option 2, you'd reference the Dragonfly-derived Cue types from `testapi` in the output schema.

Reply:
Option 2 would look like:

```
output: {
    test: testapi.#TestResource
    findings: [...testapi.#FindingResource]
    outcome: // TODO: define the shape for this one...
}
output: {
    // the concrete values you've already got here...
}
```